### DISCUSSION TITLE
version feature.

### DISCUSSION THREAD

**Discussion by mshahitxtav** [ng-icons/ng-icons#69] [Ideas]
I was wondering if version 'dropdown' can be added to the [browse-icons](https://ng-icons.github.io/ng-icons/#/browse-icons). I am working with Angular 14 and I was able to install the supported version, however, referencing back to the browse-icons page, I go and find an icon I like and find out that the icon does not exist in the version I have installed (22.4.0). Some of the icon-provider like remixicons also does not exist in version 22.4.0. It would be really helpful if we can select a version in the browse-icons page, so that we are able to see what icons providers and icons are available, and not have to constantly find out by trial and error. 

If there is way to do that currently, please point me to the right direction and disregard this post.

**Comment by ashley-hunter** [comment]
Thanks for the suggestion. Ideally, the version of the icon libraries shouldn't matter, they are essentially a collection of strings rather than anything Angular related. So hypothetically you could always use the latest versions of those packages on any version of Angular as long as the ng icon core library uses a version compatible with your version of Angular.

The Angular package format has changed in recent versions which may cause some issues with older versions of Angular currently - but you could try and see if installing the icon set library at the latest version works (I'm currently on vacation and not able to access my machine to verify myself).

Alternatively, if that doesn't work, I can use alternative tooling to build the icon set libraries to allow it to work with any version angular, so you can always use the latest.
